configure.ac

```diff
@@ -471,20 +478,23 @@ fi
 AM_CONDITIONAL([ENABLE_ICATOK], [test "x$enable_icatok" = "xyes"])
 
 dnl --- enable_ccatok
-if test "x$enable_ccatok" = "xyes"; then
+if test "x$enable_ccatok" = "xyes" -a "x$enable_pkey" = "xyes"; then
 	AC_CHECK_HEADER([asm/pkey.h], [], [
 		AC_MSG_ERROR([cca token build requested but Kernel's pkey header couldn't be found])
 	])	
 fi
 if test "x$enable_ccatok" != "xno"; then
 	enable_ccatok=yes
-	AC_CHECK_HEADER([asm/pkey.h], [], [
-		enable_ccatok=no
+	if test "x$enable_pkey" = "xyes"; then 
+		AC_CHECK_HEADER([asm/pkey.h], [], [
+			enable_ccatok=no
 	])
+	fi
 else
 	enable_ccatok=no
 fi
 AM_CONDITIONAL([ENABLE_CCATOK], [test "x$enable_ccatok" = "xyes"])
+AM_CONDITIONAL([NO_PKEY], [test "x$enable_pkey" != "xyes"])
```

Comment:
I would move this above to below the `case $target in` block. Thats where the value of enable_pkey is set.

Reply:
Done.